### DISCUSSION TITLE
Enable Brotli compression

### DISCUSSION THREAD

**Discussion by dmo9** [caprover/caprover#1380] [Q&A]
Has anyone done this in caprover before? I'd prefer to use brotli instead of gZip.

**Comment by Riches** [comment]
Yes it is possible, you need to replace the default NGINX image CapRover uses by [overriding the config](https://caprover.com/docs/troubleshooting.html#customize-config-settings)  and then add the required NGINX config.

For full steps you can follow my blog post: [Enabling Brotli content encoding on CapRover](https://www.rich.es/blog/enabling-brotli-content-encoding-on-caprover/).
